dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ORCIDHelper.java

```diff
@@ -121,6 +128,23 @@ public static HttpResponse postWorkString(String baseURL, String id, String work
         }
     }
 
+    /**
+     * This updates an existing ORCID work
+     */
+    public static HttpResponse putWorkString(String baseURL, String id, String workString, String token, String putCode) throws IOException {
+        HttpPut postRequest = new HttpPut(baseURL + id + "/work/" + putCode);
+        postRequest.addHeader("content-type", "application/vnd.orcid+xml");
+        postRequest.addHeader("Authorization", "Bearer " + token);
+        StringEntity workEntity = new StringEntity(workString);
+        postRequest.setEntity(workEntity);
+        CloseableHttpClient httpClient = HttpClientBuilder.create().build();
+        try {
```

Comment:
A bit cleaner with try catch with resources
https://stackoverflow.com/questions/21574478/what-is-the-difference-between-closeablehttpclient-and-httpclient-in-apache-http